app/views/terms-and-conditions/private-beta-2.njk

```diff
@@ -268,13 +268,17 @@
   </div>
 </div>
 
-<div class="govuk-grid-row">
+{% if showContinueButton %}
+<div class="govuk-grid-row" id="continueButton">
```

Comment:
shoudn't this be nested inside <div class="govuk-grid-column-two-thirds">?

Reply:
I did originally have it like that, but I had an issue with the ID tag in the govukbutton not being picked up by my cheerio test. So i went for just not showing the whole DIV container

Reply:
yeah, looks correct and follows this pattern - closing

<img width="424" alt="image" src="https://github.com/DEFRA/ffc-ahwr-farmer-apply/assets/118533775/d781a3af-f8bb-4bf4-9146-a18bec764d1c">